draft-add-requirements.md

```diff
@@ -55,66 +92,119 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
 document are to be interpreted as described in BCP 14 {{RFC2119}} {{!RFC8174}}
 when, and only when, they appear in all capitals, as shown here.
 
-# Discovery of associated resolvers {#first-usecase}
-
+{: #first-usecase}
+# Discovery of associated resolvers
 A client may begin with information from the attached networks, and/or resolvers known from configuration.
```

Comment:
The text is discussing auto-upgrade mechanism but should focus on the need to discover resolver in the attached network. Further, this use case is specific to Home networks and is not applicable to Enterprise networks.

Reply:
On your first point, can you contribute some text? On the second, which part of this text is incompatible with enterprise networks?

Reply:
I meant the auto-upgrade mechanism will probably work for ISPs but expecting several enterprise networks (including Small Office/Home Office (SOHO) and Small/Medium Enterprise (SME)) to work with browsers/OS to get added to the  client resolver config looks nearly impossible and daunting task.

Reply:
I don't follow why SMEs (or anyone) would need to obtain agreement from browsers and OSs. This draft is not proposing that a list of known resolvers is embedded in clients. Perhaps this is a misreading of the text; perhaps you've interpreted it to mean "A client begins with information about known encrypted networks in its configuration".  I will try to reword it to make the intent clearer.

Reply:
Done. Let me know if it still doesn't make sense.